pkg/eval/testutils.go

```diff
@@ -120,6 +120,31 @@ func (e errWithMessage) matchError(e2 error) bool {
 	return e2 != nil && e.msg == e2.Error()
 }
 
+// An errorMatcher for an ExternalCmdExit error that ignores the Pid member.
+// That is, we only match the command name and raw exit status.
+type errCmdExit struct{ v error }
+
+func (e errCmdExit) Error() string { return fmt.Sprintf("%#v", e.v) }
+
+func (want_err errCmdExit) matchError(got_err error) bool {
```

Comment:
Old habits die hard. Fixed. However, I'm ambivalent about using `e` rather than `wantErr` here. I only used `e` as the receiver var in the `Error()` definition to match the other code in this module. In this method it seems to me using `wantErr` is clearer. Nonetheless, I'll use `e` rather than `wantErr`.